src/grelu/model/blocks.py

```diff
@@ -697,22 +698,38 @@ def __init__(
         pos_dropout: float,
         attn_dropout: float,
         ff_dropout: float,
+        flash_attn: bool,
         dtype=None,
         device=None,
     ) -> None:
         super().__init__()
         self.norm = Norm("layer", in_len)
-        self.mha = Attention(
-            in_len=in_len,
-            n_heads=n_heads,
-            n_pos_features=n_pos_features,
-            key_len=key_len,
-            value_len=value_len,
-            pos_dropout=pos_dropout,
-            attn_dropout=attn_dropout,
-            dtype=dtype,
-            device=device,
-        )
+
+        if flash_attn:
+
+            print(
```

Comment:
This warning will print every time a transformer block is created - so 8 times while creating Borzoi? Maybe it would be better to make these arguments optional (e.g. key_len: Optional[int] = None) and then have the warning print only if the user tries to supply the argument with flash_attn=True?

Reply:
set up a class atrribute `flash_attn_warn` that will only warn once